src/dos/program_biostest.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2020-2021  The DOSBox Staging Team
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include <string>
+
+#include "drives.h"
+#include "program_biostest.h"
+#include "regs.h"
+
```

Comment:
can you move the program's own header first in the list (here's `midi_fluidsynth.cpp` top bit as an example):

``` c++

#include "midi_fluidsynth.h"

#include <cassert>
#include <deque>
#include <string>
#include <tuple>

#include "control.h"
#include "cross.h"
#include "fs_utils.h"
#include "mixer.h"
```